rstudio-tidyverse/rockcraft.yaml

```
@@ -0,0 +1,259 @@
+    working-dir: /home/
+    environment:
+       NB_USER: _daemon_
+       NB_UID: 1000
+       NB_GID: 0
+       NB_PREFIX: /
+       USERS_GID: 100
+       HOME: /home/_daemon_
+       HOME_TMP: /tmp_home/_daemon_
+       USER: _daemon_
+       SHELL: /bin/bash
+       LANG: en_US.UTF-8to
+       LANGUAGE: en_US.UTF-8
+       LC_ALL: en_US.UTF-8
+       PATH: /opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+       PYTHONPATH: /opt/conda/lib/python3.11/site-packages/
+       RSTUDIO_RUNTIME_DIR: "/tmp/rstudio_runtime"
+       XDG_RUNTIME_DIR: "/tmp/runtime-_daemon_"
+parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
+      dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) \
+      > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
+
+  overlay-pkgs:
+    plugin: nil
+    overlay-packages:
+      - apt-transport-https
+      - bash
+      - bzip2
+      - ca-certificates
+      - curl
+      - git
+      - gnupg
+      - gnupg2
+      - locales
+      - lsb-release
+      - nano
+      - software-properties-common
+      - tzdata
+      - unzip
+      - vim
+      - wget
+      - xz-utils
+      - zip
+
+  kubectl:
+    plugin: nil
+    stage-snaps:
+      - kubectl/1.31/stable
+    organize:
+      kubectl: bin/kubectl
+    stage:
+      - bin/kubectl
+
+  generate-locale:
+    plugin: nil
+    build-packages:
+      - locales
+    override-build: |
+      echo "en_US.UTF-8 UTF-8" > /etc/locale.gen
+      locale-gen
+      mkdir -p $CRAFT_PART_INSTALL/usr/lib/locale $CRAFT_PART_INSTALL/usr/share/i18n/locales
+      cp /usr/lib/locale/locale-archive $CRAFT_PART_INSTALL/usr/lib/locale/locale-archive
+      cp -r /usr/share/i18n/locales $CRAFT_PART_INSTALL/usr/share/i18n/locales
+
+  rstudio-tidyverse:
+    plugin: nil
+    source: https://github.com/kubeflow/kubeflow
+    source-type: git
+    source-tag: v1.10.0-rc.1
+    build-packages:
+      - lsb-release
+      - wget
+      - dpkg-sig
+      - libapparmor1
+      - libc6
+      - libclang-dev
+      - libedit2
+      - libpq5
+      - libssl-dev
+      - psmisc
+      - rrdtool
+      - sudo
+    build-environment:
+      - S6_VERSION: "3.2.0.2"
+      - MINIFORGE_VERSION: "24.11.3-0"
+      - PIP_VERSION: "24.3.1"
+      - PYTHON_VERSION: "3.11.11"
+      - RSTUDIO_VERSION: "2024.12.1-563"
+      - R_BASE_VERSION: "4.4.2"
+      - R_PNG_VERSION: "0.1_8"
+      - R_RETICULATE_VERSION: "1.40.0"
+      - R_SHINY_VERSION: "1.10.0"
+      - CONDA_DIR: "/opt/conda"
+      - PATH: "${CONDA_DIR}/bin:${PATH}"
+      - R_HOME: "${CONDA_DIR}/lib/R"
+      - R_TIDYVERSE_VERSION: "2.0.0"
+      - RS_LOG_LEVEL: "WARN"
+      - RS_LOGGER_TYPE: "stderr"
+      - MINIFORGE_ARCH: "x86_64"
+      - NB_USER: "_daemon_"
```

suggestion: Let's stick to the `jovyan` user
```suggestion
      - NB_USER: "jovyan"
```

---

Fixed